Detectors/MUON/MCH/Calibration/include/MCHCalibration/MCHChannelCalibratorAlt.h

```diff
@@ -0,0 +1,106 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef MCH_CHANNEL_CALIBRATOR_H_
+#define MCH_CHANNEL_CALIBRATOR_H_
```

Comment:
this class seem not to be used/compiled, but it has the same guards as https://github.com/AliceO2Group/AliceO2/blob/445d4cff6ca5f0385beff377b7535916a8c41c42/Detectors/MUON/MCH/Calibration/include/MCHCalibration/MCHChannelCalibrator.h#L11-L12, should be changed

Reply:
This was a test of the alternative approach I proposed at the beginning, and the files were committed by mistake. I will remove them.